rotkehlchen/chain/evm/contracts.py

```diff
@@ -263,6 +263,10 @@ def abi(self: 'EvmContracts[Literal[ChainID.ETHEREUM]]', name: 'ETHEREUM_KNOWN_A
     def abi(self: 'EvmContracts[Literal[ChainID.OPTIMISM]]', name: Literal['to', 'do', 'CTOKEN']) -> list[dict[str, Any]]:  # noqa: E501
         ...
 
+    @overload
+    def abi(self: 'EvmContracts[Literal[ChainID.POLYGON_POS]]', name: Literal['']) -> list[dict[str, Any]]:  # noqa: E501
```

Comment:
Can you also correct the Optimism one? It has a sneaky 'TODO'. And CTOKEN seems wrong. IF there are no abis we ask from there just use empty string literal there too.

But we probably should have a list per chain? Shouldn't we?

Reply:
Yeah, we have a list for ethereum. But for optimism and polygon there are no abis used atm, so I guess a list does not make sense there yet